.dockerignore

```diff
@@ -13,3 +13,4 @@
 .editorconfig
 .git*
 *.md
+docker/
```

Comment:
Oh curious, wasn't this covered by the "Ignore all subdirectories"-statement (`*/*`) above. I'll have a look.